### PR TITLE
fix secret item bug

### PR DESCRIPTION
**Pull request by jeaniewu**

### CODE REVIEW

File: Assets/Scripts/Trigger/UITrigger.cs [hunk 1]
```diff
@@ -19,37 +19,38 @@ public class UITrigger : Interact {
...
+		StartCoroutine("enableSecretMessage");
+    }
+
+	IEnumerator enableSecretMessage(){
 		manager.disablePlayer ();
 		LetterCanvas.SetActive (true);
 		isActive = true;
 		found = true;
-		GameManager.getInstance ().secretItemFound [index] = true;
+		GameManager.getInstance ().secretItemFound [index] = found;
```
Comment (jayfriso) [inline Assets/Scripts/Trigger/UITrigger.cs:38]: wait how does this change anything?

Comment (jeaniewu) [inline Assets/Scripts/Trigger/UITrigger.cs:38]: Nothing really, this one is not incorrect either

**Comment by Gowtham100** [comment]
Think this is good to go right...?

**Comment by jeaniewu** [comment]
I'm waiting for +1

**Comment by jayfriso** [comment]
:+1:

**Pull request merged**